frontend/public/components/cluster-settings/basicauth-idp-form.tsx

```diff
@@ -238,3 +252,7 @@ export type AddBasicAuthPageState = {
   inProgress: boolean;
   errorMessage: string;
 };
+
+type AddBasicAuthPageProps = {
+  t?: any;
```

Comment:
I believe this should be `TranslationFunction` imported from `i18next`.

```suggestion
  t?: TranslationFunction;
```

But hopefully it's not needed if you can use `WithTranslation` above.

Reply:
<img width="483" alt="Screen Shot 2020-10-13 at 5 15 39 PM" src="https://user-images.githubusercontent.com/895728/95917125-bd1d1080-0d77-11eb-8235-d874d433e646.png">

Reply:
It looks like @dtaylor113 was able to use `TFunction`:

https://github.com/openshift/console/pull/6928/files#diff-2c5db562a1cd010080ca3fc0f6f2277ceb8987406ab00fdda2de152470c404b9R48